src/mlpack/core/util/io.cpp

```diff
@@ -267,3 +267,50 @@ void IO::ClearSettings()
   GetSingleton().aliases = persistentAliases;
   GetSingleton().functionMap = persistentFunctions;
 }
+
+void IO::SanityChecks()
```

Comment:
I think it would be a good idea to change this to `CheckInputMatrices()` to match the name of the option.

Reply:
yes it would be better here, changed it to `CheckInputMatrices()`